dashboard/src/modules/components/SidebarComponent/index.jsx

```diff
@@ -12,27 +19,30 @@ const Menu = () => {
   const { pathname } = useLocation();
 
   const activeItem = useSelector((state) => state.sidebar.activeMenuItem);
+  const isLoggedIn = Cookies.get("isLoggedIn");
   const onSelect = (result) => {
     dispatch(setActiveItem(result.itemId));
   };
 
   useEffect(() => {
     if (pathname) {
-      const currPath = pathname.split("/").at(-1);
+      const modPathName = pathname.replace(/\/+$/, "");
+      const currPath = modPathName.split("/").at(-1);
```

Comment:
If you're going to use a regular expression, let's do this in one operation:
```
      const currPath = pathname.replace(/^.*[/]([^/]+)[/]*$/, "$1");
```
That is, take the value of `pathname`, find a sequence of non-`/` characters which is followed by zero or more `/`'s and preceded by some number of characters ending in a `/`, and return just the sequence, without the preceding or following characters.

(I think that this would be better expressed by using `match()` instead of `replace()`, but I didn't like the resulting code better, so I stuck with `replace()`.  Also, if there might not be a `/` before the value that we want for `currPath`, we just need to tweak the regex a tiny bit to make the `.*[/]` optional.)

Reply:
Got it! Updated the regex pattern